### ISSUE TITLE
Accessibility issue: Scrollable region must have keyboard access on /notebook-ci-testing/notebooks/ACS/acs_saturation_trails/acs_saturation_trails.html

### ISSUE THREAD

**Issue by dulude** [spacetelescope/notebook-ci-testing#236]
## What
An accessibility scan flagged 30 elements on https://spacetelescope.github.io/notebook-ci-testing/notebooks/ACS/acs_saturation_trails/acs_saturation_trails.html because scrollable region must have keyboard access. Learn more about why this was flagged by visiting https://dequeuniversity.com/rules/axe/4.11/scrollable-region-focusable?application=playwright.

The following elements need attention:

- `<pre id="codecell1"><span></span>The following task in the stsci.skypac package can be run with TEAL:
                                    skymatch                                    
</pre>` (selector: `#codecell1`)
- `<pre id="codecell3">` (selector: `#codecell3`)
- `<pre id="codecell4"><span></span>INFO: 18 of 36 products were duplicates. Only returning 18 unique product(s). [astroquery.mast.utils]
</pre>` (selector: `#codecell4`)
- `<pre id="codecell5"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/jdg302cvq_flc.fits to ./mastDownload/HST/jdg302cvq/jdg302cvq_flc.fits ...
</pre>` (selector: `#codecell5`)
- `<pre id="codecell7"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/jdg302ctq_flc.fits to ./mastDownload/HST/jdg302ctq/jdg302ctq_flc.fits ...
</pre>` (selector: `#codecell7`)
- `<pre id="codecell9"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/jdg301byq_flc.fits to ./mastDownload/HST/jdg301byq/jdg301byq_flc.fits ...
</pre>` (selector: `#codecell9`)
- `<pre id="codecell11"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/jdg301bwq_flc.fits to ./mastDownload/HST/jdg301bwq/jdg301bwq_flc.fits ...
</pre>` (selector: `#codecell11`)
- `<pre id="codecell13"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/jdg302cnq_flc.fits to ./mastDownload/HST/jdg302cnq/jdg302cnq_flc.fits ...
</pre>` (selector: `#codecell13`)
- `<pre id="codecell15"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/jdg302clq_flc.fits to ./mastDownload/HST/jdg302clq/jdg302clq_flc.fits ...
</pre>` (selector: `#codecell15`)
- `<pre id="codecell17"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/jdg301ccq_flc.fits to ./mastDownload/HST/jdg301ccq/jdg301ccq_flc.fits ...
</pre>` (selector: `#codecell17`)
- `<pre id="codecell19"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/jdg301ceq_flc.fits to ./mastDownload/HST/jdg301ceq/jdg301ceq_flc.fits ...
</pre>` (selector: `#codecell19`)
- `<pre id="codecell21"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/jdg301c4q_flc.fits to ./mastDownload/HST/jdg301c4q/jdg301c4q_flc.fits ...
</pre>` (selector: `#codecell21`)
- `<pre id="codecell23"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/jdg301c6q_flc.fits to ./mastDownload/HST/jdg301c6q/jdg301c6q_flc.fits ...
</pre>` (selector: `#codecell23`)
- `<pre id="codecell25"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/jdg302czq_flc.fits to ./mastDownload/HST/jdg302czq/jdg302czq_flc.fits ...
</pre>` (selector: `#codecell25`)
- `<pre id="codecell27"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/jdg302cxq_flc.fits to ./mastDownload/HST/jdg302cxq/jdg302cxq_flc.fits ...
</pre>` (selector: `#codecell27`)
- `<pre id="codecell29"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/jdg303d7q_flc.fits to ./mastDownload/HST/jdg303d7q/jdg303d7q_flc.fits ...
</pre>` (selector: `#codecell29`)
- `<pre id="codecell31"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/jdg303d5q_flc.fits to ./mastDownload/HST/jdg303d5q/jdg303d5q_flc.fits ...
</pre>` (selector: `#codecell31`)
- `<pre id="codecell33"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/jdg303dbq_flc.fits to ./mastDownload/HST/jdg303dbq/jdg303dbq_flc.fits ...
</pre>` (selector: `#codecell33`)
- `<pre id="codecell35"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/jdg303d9q_flc.fits to ./mastDownload/HST/jdg303d9q/jdg303d9q_flc.fits ...
</pre>` (selector: `#codecell35`)
- `<pre id="codecell37"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/jdg303dnq_flc.fits to ./mastDownload/HST/jdg303dnq/jdg303dnq_flc.fits ...
</pre>` (selector: `#codecell37`)
- `<pre id="codecell39"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/jdg303dlq_flc.fits to ./mastDownload/HST/jdg303dlq/jdg303dlq_flc.fits ...
</pre>` (selector: `#codecell39`)
- `<pre id="codecell47">` (selector: `#codecell47`)
- `<pre id="codecell79">` (selector: `#codecell79`)
- `<pre id="codecell94"><span></span>/home/runner/micromamba/envs/ci-env/lib/python3.11/site-packages/numpy/_core/fromnumeric.py:867: UserWarning: Warning: &apos;partition&apos; will ignore the &apos;mask&apos; of the MaskedArray.
  a.partition(kth, axis=axis, kind=kind, order=order)
</pre>` (selector: `#codecell94`)
- `<pre id="codecell99"><span></span>/home/runner/micromamba/envs/ci-env/lib/python3.11/site-packages/numpy/_core/fromnumeric.py:867: UserWarning: Warning: &apos;partition&apos; will ignore the &apos;mask&apos; of the MaskedArray.
  a.partition(kth, axis=axis, kind=kind, order=order)
</pre>` (selector: `#codecell99`)
- `<pre id="codecell104">` (selector: `#codecell104`)
- `<pre id="codecell105">` (selector: `#codecell105`)
- `<pre id="codecell107">` (selector: `#codecell107`)
- `<pre id="codecell109">` (selector: `#codecell109`)
- `<pre id="codecell111">` (selector: `#codecell111`)

To fix this, ensure elements that have scrollable content are accessible by keyboard in safari.

Specifically:

Fix any of the following:
-   Element should have focusable content
-   Element should be focusable

## Acceptance Criteria
- [ ] The specific violation reported in this issue is no longer reproducible.
- [ ] The fix MUST meet WCAG 2.2 guidelines OR the accessibility standards specified by the repository or organization.
- [ ] A test SHOULD be added to ensure this specific violation does not regress.
- [ ] This PR MUST NOT introduce any new accessibility issues or regressions.
